test/CMakeLists.txt

```diff
@@ -654,7 +654,7 @@ set(IMPLICITGEMM_MLIR_ARGS_B ${IMPLICITGEMM_ARGS} --verbose --disable-forward --
 set(IMPLICITGEMM_MLIR_ARGS_W ${IMPLICITGEMM_ARGS} --verbose --disable-forward --disable-backward-data)
 
 # Note: OpenCL Debug + Codecov test stage taking longer time than a smoke test should therefore disabling that scenario
-if ((NOT CMAKE_BUILD_TYPE MATCHES Debug) OR (NOT ${CODECOV_TEST}))
+if ((NOT uppercase CMAKE_BUILD_TYPE MATCHES "DEBUG") OR (NOT ${CODECOV_TEST}))
```

Comment:
I am unable to find this syntax in CMake docs. You can try `string(TOUPPER ${VARNAME} VARNAME)` before if.

Reply:
Done in f3dd6f4